kb/2025-03-14/iso_boot_fails_with_sbat_error.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Harvester ISO boot fails with SBAT error
+description: 'How to work around "Verifying shim SBAT data failed: Security Policy Violation" error when booting the Harvester installer ISO'
+slug: iso_boot_fails_with_sbat_errror
+authors:
+  - name: Tim Serong
+    title: Master Software Engineer
+    url: https://github.com/tserong
+    image_url: https://github.com/tserong.png
+tags: [installation, operating system, secure boot]
+hide_table_of_contents: false
+---
+
+When installing Harvester on a host that previously had an operating system installed and has UEFI secure boot enabled, the ISO image may fail to boot with the following error:
+
+```
+Verifying shim SBAT data failed: Security Policy Violation
+Something has gone seriously wrong: SBAT self-check failed: Security Policy Violation
+```
+
+This issue is known to occur if openSUSE Leap 15.5 or Harvester v1.3.1 or newer were installed previously. It is also likely to occur if a recent version of Windows was installed previously, and may occur if other Linux variants were installed previously.
```

Comment:
"This issue is known to occur if openSUSE Leap 15.5 or Harvester v1.3.1 or newer were installed previously." - is it a newer version of both openSUSE Leap and Harvester or just a newer version of Harvester?

Reply:
both, actually

Reply:
..should I instead say "...openSUSE Leap 15.5 or newer, or Harvester v1.3.1 or newer..."?